site/public/js/user-profile.js

```diff
@@ -149,6 +160,51 @@ function updateUserPronouns(e) {
     }
 }
 
+// update user name order and display name order option
+function updateDisplayNameOrder(e) {
+    // update user name order
+    e.preventDefault();
+    const displayNameOrder = $('#display-name-order-change');
+    displayNameOrderLast = displayNameOrder.val();
+
+    const data = new FormData();
+    // eslint-disable-next-line no-undef
+    data.append('csrf_token', csrfToken);
+    data.append('display-name-order', displayNameOrder.val());
+    const url = buildUrl(['user_profile', 'change_display_name_order']);
+    $.ajax({
+        url,
+        type: 'POST',
+        data,
```

Comment:
This is very minor request, style-wise lets keep it consistent with rest of the codebase
```suggestion
        url: url,
        type: 'POST',
        data: data,
```